### ISSUE TITLE
[BUG] A11y issue - Alt text on social buttons

### ISSUE THREAD

**Issue by InHuOfficial** [eddiejaoude/eddiejaoude#113]
## Issue
The Twitter, Discord and YouTube social sharing buttons have no alt text, meaning that when someone using assistive technology focuses them they will possibly get read the image `src` as the link text in some screen reader / browser combos.

![screenshot of social share buttons on Eddie's profile](https://user-images.githubusercontent.com/81360444/154923434-644987a5-e5bd-4b78-ad39-af852562ad1b.png)

## Fix
Add appropriate `alt` text to each image so the links make sense.

### Sample Markdown / Markup
I have added some suggested `alt` descriptions in the markup below.

Additionally I removed the trailing slash on `<img>` tags as not needed (optional fix depending on your style preferences)

``` html
<p align="center">
  <a href="http://twitter.com/eddiejaoude">
    <img src="https://img.shields.io/twitter/follow/eddiejaoude?label=Twitter&logo=twitter&style=for-the-badge&color=blue" alt="Twitter, Eddie Jaoude">
  </a>
  <a href="https://discord.com/invite/jZQs6Wu">
    <img src="https://img.shields.io/discord/699608417039286293?logo=discord&style=for-the-badge&color=blue" alt="EddieHub Discord">
  </a>
  <a href="http://youtube.com/eddiejaoude?sub_confirmation=1">
    <img src="https://img.shields.io/youtube/channel/subscribers/UC5mnBodB73bR88fLXHSfzYA?style=for-the-badge&logo=youtube&label=Youtube&color=blue" alt="YouTube for Eddie Jaoude">
  </a>
</p>
```

**Comment by nirban256** [comment]
Hey can i work on this issue?

**Comment by eddiejaoude** [comment]
@nirban256 sure 👍

**Comment by nirban256** [comment]
please check the pull request for this issue @eddiejaoude

**Comment by eddiejaoude** [comment]
PR merged 👍
